src/tcgwars/logic/impl/gen3/TeamRocketReturns.groovy

```diff
@@ -2542,17 +2542,17 @@ public enum TeamRocketReturns implements LogicCardInfo {
           }
         };
       case R_ENERGY_95:
-        return specialEnergy (this, [[]]) {
+        return specialEnergy (this, [[C]]) {
```

Comment:
Can I ask the reasoning for adding the default colorless value?

Reply:
Hmm I didn't make those changes, this branch must be out of date. Let me start a new one.